### PR TITLE
Azure DevOps documentation

### PR DESCRIPTION
**Pull request by bjcmit**
Added MLOps with Azure DevOps documentation along with fixing some quoted strings

### CODE REVIEW

File: docs/cd-seldon.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ In the kaizentm/kubemlops repository when a model is deployed, which is performe
 ## Access CD pipeline via GitHub
 
 1. Clicking on your pull request commit in GitHub actions, you will get the CI/Training pipeline steps. ![GitHub Commit CI](./diagrams/actions-commit-ci.png)
-2. Clicking on “Model Training” on the left side and then “View more details”, you get all the GitHub actions associated with your pull request. ![GitHub PR Summary](./diagrams/pr-summary.png)
-3. Clicking on “CD” on the left side, you get the details on failed and finished steps.
+2. Clicking on `Model Training` on the left side and then `View more details`, you get all the GitHub actions associated with your pull request. ![GitHub PR Summary](./diagrams/pr-summary.png)
+3. Clicking on `CD` on the left side, you get the details on failed and finished steps.
```
Comment (dtzar) [inline docs/cd-seldon.md:13]: We need to tell people to replace the azure-specific scoring URI in the test deployed model section.

File: README.md [hunk 2]
```diff
@@ -30,16 +30,20 @@ Before you get started, make sure you have the following prerequisites:
...
+This documentation helps you get started with the sample from infrastructure setup to deployment of the model.
+
 - [Kubeflow Installation on AKS](./docs/Kubeflow-install.md)
 - [Securing Kubeflow on AKS](./docs/Kubeflow-secure.md)
 - MLOps with GitHub
   - [CI/Pull Request](./docs/ci-pullrequest.md)
   - [CI/Training](./docs/ci-training.md)
   - [CD/Seldon](./docs/cd-seldon.md)
+- MLOps with Azure DevOps
+  - [Pull Request](./docs/azdo-pr.md)
```
Comment (dtzar) [inline README.md:44]: This feels like too much separation of docs.  IMO we should have a PR/CI/CD doc for Azure DevOps and then one more PR/CI/CD doc for GitHub.

File: docs/cd-seldon.md (hunk 1 above)
Comment (dtzar) [inline docs/cd-seldon.md:13]: Please tell people how to find/update the scoring URL in the Test deployed model below.

Comment (bjcmit) [reply docs/cd-seldon.md:13]: This looks very similar to your first comment. What is the difference?

File: docs/mlops-github.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+1. Clicking on your pull request commit in GitHub actions, you will get the CI/Training pipeline steps. ![GitHub Commit CI](./diagrams/actions-commit-ci.png)
+2. Clicking on `Model Training` on the left side and then `View more details`, you get all the GitHub actions associated with your pull request. ![GitHub PR Summary](./diagrams/pr-summary.png)
+3. Clicking on `CD` on the left side, you get the details on failed and finished steps.
+
+### Test deployed model
+
+- Test with an image of a burrito:
+
+```bash
+curl -H "Content-Type: application/json" -d '{"image":"https://www.exploreveg.org/files/2015/05/sofritas-burrito.jpeg"}' {Scoring_URI}
```
Comment (dtzar) [inline docs/mlops-github.md:67]: Just add in how to find the scoring URI and then LGTM

**Pull request merged**